.github/images/Dockerfile

```diff
@@ -1,14 +1,11 @@
 # image-name: ghcr.io/dena/unity-meta-check/unity-meta-check-gh-action
-FROM golang:1.19-buster as builder
-WORKDIR /go/src/unity-meta-check
-COPY . .
-RUN make out/gh-action-linux-amd64 && mv ./out/gh-action-linux-amd64 ./out/gh-action
+FROM ghcr.io/dena/unity-meta-check/unity-meta-check-builder:latest as builder
 
-FROM debian:buster-slim
+FROM debian:bookworm-slim
```

Comment:
[Proposal] I suggest to use distroless image for base image. The distroless images are lightweight and secure.
https://github.com/GoogleContainerTools/distroless

I tried changing the image for a test and the image size was reduced to about 1/5 of the original size. This reduces CI run time.
```diff
diff --git a/.github/images/Dockerfile b/.github/images/Dockerfile
index 0aa0b27..c74c8d1 100644
--- a/.github/images/Dockerfile
+++ b/.github/images/Dockerfile
@@ -1,11 +1,7 @@
 # image-name: ghcr.io/dena/unity-meta-check/unity-meta-check-gh-action
 FROM ghcr.io/dena/unity-meta-check/unity-meta-check-builder:latest as builder

-FROM debian:bookworm-slim
-RUN apt-get update \
-       && apt-get install --yes --no-install-recommends git ca-certificates \
-       && apt-get clean \
-       && rm -rf /var/lib/apt/lists/*
+FROM gcr.io/distroless/base-debian12
 COPY --from=builder /go/src/unity-meta-check/out/gh-action /usr/bin/
 ENTRYPOINT ["gh-action"]
 CMD ["-help"]
```

```console
$ docker images | grep unity-meta
unity-meta-check-distroless                                                                         latest                                     8eb555c488c6   6 minutes ago   9.74MB
unity-meta-check-debian                                                                             latest                                     ee729b0692e0   7 minutes ago   201MB

$ docker run --rm -it unity-meta-check-distroless
usage: unity-meta-check-gh-action -inputs-json <json>
  -inputs-json string
        JSON string of "inputs" context value of GitHub Actions
  -version
        print version

```

Reply:
Sorry, GitHub recommended using debian-based images.

> It's recommended to use Docker images based on the [Debian](https://www.debian.org/) operating system.

https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions